### PR TITLE
Restruct APIs in v2

### PR DESCRIPTION
**Pull request by XuanYang-cn**
Signed-off-by: yangxuan <xuan.yang@zilliz.com>

### CODE REVIEW

**Comment by longjiquan** [comment]
LGTM
/assign @czs007

File: pymilvus/v2/apis.py [hunk 1]
```diff
@@ -10,132 +10,192 @@
...
+    # TODO
+    def get_server_version(timeout: float=None, **kwargs) -> str:
+        pass
+
+    # TODO
+    def insert(collection_name: str, column_based_entities: List[list], partition_name: str="_default", timeout: float=None, **kwargs):
+        pass
+
+    # TODO
+    def insert_by_rows(collection_name: str, row_based_entities: List[list], partition_name: str="_default", timeout: float=None, **kwargs):
```
Comment (czs007) [inline pymilvus/v2/apis.py:148]: insert_rows
you can fix this in next pr.

File: pymilvus/v2/apis.py [hunk 2]
```diff
@@ -10,132 +10,192 @@
...
+    # TODO
+    def upsert(timeout: float=None):
+        pass
+
+    # TODO
+    def delete(collection_name: str, pks: List[Union[int, str]], partition_name: str="_default", timeout: float=None, **kwargs):
+        pass
+
+    # TODO
+    def delete_by_expression(collection_name: str, expression: str, partition_name: str="_default", timeout: float=None, **kwargs):
```
Comment (czs007) [inline pymilvus/v2/apis.py:164]: delete_by_expr
you can fix this in next pr.

**Pull request merged**